libraries/AC_WPNav/AC_Loiter.cpp

```diff
@@ -305,12 +305,22 @@ void AC_Loiter::calc_desired_velocity(float nav_dt)
 
     // Limit the velocity to prevent fence violations
     // TODO: We need to also limit the _desired_accel
```

Comment:
We can remove this TODO comment now I think.

Reply:
fixed and added a few more comments to loiter